packages/wallets/src/evm/chains.ts

```diff
@@ -0,0 +1,22 @@
+import { BlockchainIncludingTestnet as Blockchain } from "@crossmint/common-sdk-base";
+
+const SMART_WALLET_TESTNET_CHAINS = [
+    Blockchain.BASE_SEPOLIA,
+    Blockchain.POLYGON_AMOY,
+    Blockchain.OPTIMISM_SEPOLIA,
+    Blockchain.ARBITRUM_SEPOLIA,
+];
+
+const SMART_WALLET_MAINNET_CHAINS = [Blockchain.BASE, Blockchain.POLYGON, Blockchain.OPTIMISM, Blockchain.ARBITRUM];
```

Comment:
don't we support more of them?

Reply:
added Story in https://github.com/Crossmint/crossmint-sdk/pull/956/commits/7594825780a70acaa840a8913a1aaa03f18d5ca3

to add Mode we first need to update the chain definitions in `@crossmint/common-sdk-base`.

Reply:
Is it not possible to reuse the supported chains that we have in the openapi schema? It'd be nice to only have to update it in one place and not have to remember to do it here as well